docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,102 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. Before
+converting into IR PyTorch model need to be exported into ONNX. In order to that follow the steps below.
```

Comment:
Before converting into IR PyTorch model need to be exported into ONNX. In order to that follow the steps below. --->
Follow the steps below to export a PyTorch* model into ONNX* before converting it to IR: